include/ostatus.php

```diff
@@ -238,12 +245,14 @@ public static function import($xml,$importer,&$contact, &$hub) {
 
 		$gub = "";
 		$hub_attributes = $xpath->query("/atom:feed/atom:link[@rel='hub']")->item(0)->attributes;
-		if (is_object($hub_attributes))
-			foreach($hub_attributes AS $hub_attribute)
+		if (is_object($hub_attributes)) {
+			foreach($hub_attributes AS $hub_attribute) {
```

Comment:
I can generally change it in the whole file. Then I don't need to make another PR for the same file and rewrite step.